Aptoide/aptoide-client-v8
# [MOB-783] New search results

**What does this PR do?**

   This PR aims at implementing the new search results view. Also took the chance to remove some unused resources and do some refactor on the search view holder.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] SearchResultViewHolder.kt

**How should this be manually tested?**

Perform a search and check that the new layout is implemented. Also confirm everything is ok with the ads. Check different possible points of failure related with the layout.

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-783](https://aptoide.atlassian.net/browse/MOB-783)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
